Documentation/ApiOverview/Logging/Configuration/Index.rst

```diff
@@ -50,6 +50,16 @@ The Log Writer configuration is read from the subkey :code:`writerConfiguration`
 
 The above configuration applies to **all** log entries of level "ERROR" or above.
 
+.. important::
+
+    Since TYPO3 v9 the default folder for log files is `typo3temp/var/log`. In v8 it was `typo3temp/var/logs`
+    and older versions used `typo3temp/logs`.
+
+.. important::
+
+    Since TYPO3 v9 it is possible and (a good practice) to store temporary files (`typo3temp/var`) outside the document root).
```

Comment:
Since TYPO3 v9 it is possible (and a good practice) to store temporary files ...